_component_design/key-value-list.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Key-value pairs
+title: Key-value List
```

Comment:
Should this be updated to "Description List"? 

The implementation uses `<dl>` which is the ["Description List element"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl).

Reply:
Yep.  I went through and updated all the design docs to match the component name in HelixUI -- with the exception of Option Tile and Button Bar